iidm/iidm-api/src/main/java/com/powsybl/iidm/network/util/SV.java

```diff
@@ -348,6 +268,9 @@ private SV otherSide(LinkData.BranchAdmittanceMatrix adm) {
             s = flowAtEnd1(adm, v, v2);
             otherSide = Branch.Side.ONE;
         }
+        if (Double.isNaN(q) || Double.isNaN(u)) { // DC approximation
```

Comment:
@zamarrenolm and @marqueslanauja this approximation is totally coherent with what we are doing in open-loadflow for DC approximation.

Reply:
Only a test is missing.

Reply:
I think we must change the alpha (`a`) on the other side. I implemented and committed the necessary code to calculate the alpha according with the DC loadflow inside the open-Loadflow. I also added unitary tests.